guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,135 @@
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> * Multiple causes with the same remedy
+> * A single cause with more than one remedy
+> * A remedy with no known cause
+> * A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+> [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+*Klaudia is preparing the new diagram style, see <https://github.com/kyma-project/community/issues/542>.*
+
+### Screenshots
+
+Wherever possible, present screenshots as SUI (simplified user interfaces). Basically, this means to blur out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Documentation Structure and Target Groups
+
+On the Kyma website, we’ll have five main tabs containing multiple documents each, plus a glossary.
+
+### Overview
+
+Target Group: Decision Makers (Tech Leads) and newbies.
+Contains a quick overview of the idea behind Kyma, a diagram of the main areas with a brief explanation, and, as needed, longer documents going into the details of each main area.
+
+### Getting Started
+
+Target Group: Software Developers who quickly want to see what they can do with Kyma.
+Contains a guide/tutorial designed by PB.
+
+### Deep Dive/Tutorials
+
+Target Group: Software Developers leveraging all the Kyma functionalities.
+Under this tab, there will be subtabs according to main areas (except UI – Busola and CLI are mentioned as needed within the instructions of the respective main area.)
+
+### Operations
+
+Target Group: Admins/Operators who make sure the Kyma cluster is configured as needed and keeps running in a healthy and secure way.
+Contains installation and configuration instructions, backup info, troubleshooting guides…
+
+### Technical References
+
+Target Group: Users who want need to look up specific detailed information.
+Contains the architecture diagrams, configuration charts, etc,; no explanation of concepts or instructions.
+
+### Glossary
+
+Target Group: Anyone who wants to look up terms they’re not familiar with.
+Explains basic terms, with a focus on terms specific to Kyma.
+
+## Content Source
+
+We write the content in [Markdown](https://daringfireball.net/projects/markdown/) and store it in [Git](https://git-scm.com/) repositories.
```

Comment:
I'd also add `The assumed reader's knowledge` and `Release notes` sections that were also included in the old content strategy.

Reply:
Good point!! Thank you.

Should we include `Release Notes` as an information type? It's got a defined structure that's doesn't really the existing info types.

Reply:
I think Klaudia meant [this](https://github.com/kyma-project/community/blob/main/guidelines/content-guidelines/01-content-strategy.md#the-assumed-readers-knowledge) section.

Reply:
I added that!